MSAL/src/native_auth/controllers/credentials/MSALNativeAuthCredentialsController.swift

```diff
@@ -88,7 +88,7 @@ final class MSALNativeAuthCredentialsController: MSALNativeAuthTokenController,
                       userAccountResult: MSALNativeAuthUserAccountResult) async -> RefreshTokenCredentialControllerResponse {
```

Comment:
we can get rid of this "refreshToken" method. 
Can you please remove it also from the `MSALNativeAuthCredentialsControlling` interface?

Reply:
Yes, please :)

Reply:
Done.